channels/id.m3u

```diff
@@ -28,21 +28,21 @@ http://iamchannel.org:1935/tes/1/playlist.m3u8
 #EXTINF:-1 tvg-id="IndonesiaChannel.id" tvg-country="ID" tvg-language="" tvg-logo="" group-title="",Indonesia Channel (720p) [Not 24/7]
 http://wowzaprod236-i.akamaihd.net/hls/live/1019903/7dd4cf51/playlist.m3u8
 #EXTINF:-1 tvg-id="Indosiar.id" tvg-country="ID" tvg-language="Indonesian" tvg-logo="" group-title="General",Indosiar [Offline]
-https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.vidio.com/live/205-indosiar
+https://iptv.dc-team.com/query?url=https://www.vidio.com/live/205-indosiar
```

Comment:
Not sure if it's just me, but I'm getting 504's when trying to access these Vidio links (either from browser, VLC or ffplay)